modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -250,7 +250,8 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
         std::vector<size_t> pads_begin, pads_end;
         CV_DEPRECATED_EXTERNAL Size kernel, stride, pad;
         CV_DEPRECATED_EXTERNAL int pad_l, pad_t, pad_r, pad_b;
-        bool globalPooling;
+        CV_DEPRECATED_EXTERNAL bool globalPooling; //!< Flag is true if at least one of the axes is global pooled.
```

Comment:
Due we are going to use this flag as well, let's remove `CV_DEPRECATED_EXTERNAL`